.gitignore

```diff
@@ -1 +1,4 @@
 dump.rdb
+.idea/
+Gopkg.*
+vendor/
```

Comment:
Is it possible to remove these lines from the `.gitignore`? Because they are either IDE specific or depending on other PRs 😸